src/features/auth/signup/AfterSignupEmailValidationBuffer.tsx

```diff
@@ -49,17 +50,23 @@ export function AfterSignupEmailValidationBuffer() {
       { accessToken: response.accessToken, refreshToken: response.refreshToken },
       'fromSignup'
     )
-    if (!response.idCheckToken || !settings?.allowIdCheckRegistration) {
+
+    try {
+      const me = await api.getnativev1me()
+      // TODO: clean nextBeneficiaryValidationStep is not yet available, this will prevent a breaking change, see with Dimitri for detail
```

Comment:
this will prevent a breaking change since api v138 sends redirection info in /me route instead of /validate_email response